src/fsdocs-tool/BuildCommand.fs

```diff
@@ -2115,3 +2119,13 @@ type WatchCommand() =
 
     [<Option("port", Required = false, Default = 8901, HelpText = "Port to serve content for http://localhost serving.")>]
     member val port = 8901 with get, set
+
+    [<Option("contenthost", Required = false, HelpText = "URI root to inject in static content.")>]
+    member val contenthost = "" with get, set
+
+    override x.static_content_host_option =
```

Comment:
This code violates some analyzers:

![image](https://github.com/fsprojects/FSharp.Formatting/assets/2621499/d46ef6a0-dcf7-4057-8d9b-559037684dfa)

Run `dotnet fsi build.fsx -- -p Verify` for more info.

Reply:
Thanks for the tip with `-p Verify`! That's helpful. 

Though the output is quite verbose with results from the other projects. It also seems to create diffs in the `tests/` directory for me. Not sure if that's intentional.